InternetHospital.BusinessLogic/Models/PatientDetailedModel.cs

```diff
@@ -0,0 +1,15 @@
+﻿using InternetHospital.DataAccess.Entities;
+using System;
+
+namespace InternetHospital.BusinessLogic.Models
+{
+    public class PatientDetailedModel
+    {
+        public string FirstName { get; set; }
+        public string SecondName { get; set; }
+        public string ThirdName { get; set; }
+        public string PhoneNumber { get; set; }
+        public DateTime? BirthDate { get; set; }
+        public IllnessHistoryModel[] IllnessHistory { get; set; }
```

Comment:
Is it important to use Array instead of IEnumerable<T>?

Reply:
No, set to IEnumerable